src/client/dfuse/dfuse.h

```diff
@@ -1017,6 +1024,9 @@ struct dfuse_inode_entry {
 
 struct active_inode {
 	d_list_t               chunks;
+	d_list_t               open_reads;
+	bool                   seen_eof;
+	size_t                 file_size;
```

Comment:
These two are not used in this PR, I'll remove them on re-push but I don't want to loose testing.